PowerValidatedSolutions.psd1

```diff
@@ -11,7 +11,7 @@
     RootModule = 'PowerValidatedSolutions.psm1'
     
     # Version number of this module.
-    ModuleVersion = '2.8.0.1039'
+    ModuleVersion = '2.8.0.1040'
```

Comment:
This build number will need to be bumped again when this PR is ready to be merged